src/components/Tokens/state.ts

```diff
@@ -1,18 +1,17 @@
 import { SupportedChainId } from 'constants/chains'
-import { TimePeriod } from 'graphql/data/Token'
+import { TokenSortMethod } from 'graphql/data/TopTokens'
+import { TimePeriod } from 'graphql/data/util'
 import { atom, useAtom } from 'jotai'
 import { atomWithReset, atomWithStorage, useAtomValue } from 'jotai/utils'
 import { useCallback, useMemo } from 'react'
 
-import { Category, SortDirection } from './types'
-
 export const favoritesAtom = atomWithStorage<string[]>('favorites', [])
 export const showFavoritesAtom = atomWithStorage<boolean>('showFavorites', false)
 export const filterStringAtom = atomWithReset<string>('')
 export const filterNetworkAtom = atom<SupportedChainId>(SupportedChainId.MAINNET)
 export const filterTimeAtom = atom<TimePeriod>(TimePeriod.DAY)
-export const sortCategoryAtom = atom<Category>(Category.marketCap)
-export const sortDirectionAtom = atom<SortDirection>(SortDirection.decreasing)
+export const sortMethodAtom = atom<TokenSortMethod>(TokenSortMethod.TOTAL_VALUE_LOCKED)
+export const sortAscendingAtom = atom<boolean>(false)
```

Comment:
why was this switching to `sortAscendingAtom`? 

If it's removed, I think we should probably drop the prior type used here from the codebase if we don't need it anymore. (`SortDirection`)

Reply:
I changed to a boolean for simplicity sake since there will only ever be two sorting directions. I believe I did remove SortDirection as well

Reply:
Ah, you're right. Didn't see the deleted file.